test/library/standard/Map/mapOfArray.skipif

```diff
@@ -1,3 +1,14 @@
-# This .future is sometimes passing for multilocale/no-local
-CHPL_COMM!=none
-COMPOPTS <= --no-local
+#!/usr/bin/env python
+
+from __future__ import print_function
+import os
+
+# This .future needs valgrind or an address sanitizer run on it
+
+sanitize = os.getenv('CHPL_SANITIZE_EXE', 'none')
+vgrnd = os.getenv('CHPL_TEST_VGRND_EXE', 'none')
+
+if 'address' == sanitize or 'on' == vgrnd:
```

Comment:
`CHPL_SANITIZE_EXE` supports a comma separated list of sanitizers (though we haven't tested any beyond asan), so I'd probably do `if 'address' in sanitize` (i.e. I wouldn't want to run this test just because leak sanitizer is enabled or something, but I would if address and leak sanitizer are enabled.)

Reply:
"Don't bug Elliot, it's basically just a modified version of what he said" <- me, a fool